DataFormats/Detectors/TRD/include/DataFormatsTRD/Digit.h

```diff
@@ -83,7 +98,7 @@ class Digit
   }
 
  private:
-  std::uint16_t mDetector{0}; // detector, the chamber [0-539]
+  std::uint16_t mDetector{0}; // detector, the chamber [0-539] ; detector value is in 0-11 [0-1023], 12-15 is the phase of the trigger from the digit hc header.
```

Comment:
could you make this comment a bit clearer please? I am confused by the 0-1023. Maybe you can put explicitly which bits are used for what and the valid ranges for both.

Reply:
Also, please increase ClassDef version

Reply:
done